### PR TITLE
Add Linux Containers

### LINKED ISSUE #2899: LXC
No duplicates

**Name:** LXC / LXC Containers
**Website:** https://linuxcontainers.org/
**Alexa rank:** [~85.9k](https://www.alexa.com/siteinfo/linuxcontainers.org)
**Official resources for icon and color:**
  
**SVG Icon:** https://linuxcontainers.org/static/img/containers.svg
**Color:** `#333333`

### PR DESCRIPTION
**Pull request by PeterShaggyNoble**
Linux Containers

**Issue:** Closes #2899

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Cleaning up the imperfection mentioned by Eric in the original issue caused the diagonals to shift slightly.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -3410,6 +3410,11 @@
             "hex": "FCC624",
             "source": "http://www.linuxfoundation.org/about/about-linux"
         },
+        {
+            "title": "Linux Containers",
+            "hex": "333333",
+            "source": "https://discuss.linuxcontainers.org/t/official-lxc-lxd-svg-icon/7346"
```
Comment (runxel) [inline _data/simple-icons.json:3416]: Why not linking to https://linuxcontainers.org/static/img/containers.svg?

Comment (adamrusted) [reply _data/simple-icons.json:3416]: Rebuilding from source and removing some of the weird paths, I got this:

linuxcontainers

```svg
M10.15,8.98l1.647,0.951l-5.756,3.345l0.005-1.911L10.15,8.98z M11.873,7.979L10.32,8.881l1.548,0.893
	L11.873,7.979z M6.028,5.33L6.025,6.4l0.543,0.316l3.602,2.079l1.632-0.948L9.612,6.568L6.381,4.681L6.03,4.478L6.028,5.33z
	 M6.1,4.347l0.359,0.209l6.321,3.65l5.258,3.037l5.858-3.405l-11.94-6.895L6.1,4.347z M12.102,16.949l-0.005,1.924l5.858-3.404
	l0.005-1.924L12.102,16.949z M12.025,7.94L12.02,9.862l5.94,3.428l0.005-1.92L12.025,7.94z M1.895,8.885l4.075,2.352l4.031-2.342
	L5.926,6.542L1.895,8.885z M24,7.982l-5.858,3.404l-0.015,3.982l5.858-3.404L24,7.982z M11.952,18.022l0.003-1.073L7.6,14.436
	l-1.565-0.903v0.001l-0.939-0.542l-5.081-2.932L0.01,11.979l11.94,6.895L11.952,18.022z M17.887,13.417l-5.922-3.411l-5.853,3.401
	l5.917,3.414L17.887,13.417z M23.959,12.179l-11.862,6.864l-0.01,4.013l11.863-6.894L23.959,12.179z M11.944,21.27l0.005-2.227
	L0.01,12.148L0,16.162l11.94,6.895L11.944,21.27z M0.021,9.802l1.579-0.917L0.025,7.976L0.021,9.802z M5.853,6.412l0.024-1.636
	l0.001-0.296L0.099,7.848l1.647,0.951L5.853,6.412z M5.894,11.363L1.749,8.97l-0.46,0.267L0.094,9.932l5.795,3.345L5.894,11.363z
```

Feel free to use it, or I can open a PR for it?

**Comment by adamrusted** [comment]
Just pushed path mentioned in https://github.com/simple-icons/simple-icons/pull/3155#discussion_r549422254.

File: _data/simple-icons.json (hunk 1 above)
Comment (PeterShaggyNoble) [reply _data/simple-icons.json]: @runxel, we try to link to a page rather than a file whenever possible as pages are less prone to link-rot.

Comment (adamrusted) [reply _data/simple-icons.json]: @PeterShaggyNoble - the source of their website, including this SVG, is available at https://github.com/lxc/linuxcontainers.org/blob/29d3299ddf8718099b6de1464570fbbadbaabecb/static/img/containers.svg

Comment (PeterShaggyNoble) [reply _data/simple-icons.json]: Yep, definitely, @adamrusted :+1: Just pushed that change.

**Comment by adamrusted** [comment]
@PeterShaggyNoble if you're happy with the edited path I included https://github.com/simple-icons/simple-icons/pull/3155/commits/241197eeb79ee59f4aa9c0775b2315985ae5544b - then feel free to merge this, unless you'd like another @simple-icons/maintainers to give this a look over first.

**Comment by PeterShaggyNoble** [comment]
Yep, if nobody else gets to it in the meantime, I'll have a look at it Monday. After a few months away, I used last week to get caught up and am using this week to remind myself of all our - and my - processes before I dive back into doing full reviews of PRs.

**Comment by PeterShaggyNoble** [comment]
This looks good to me, @adamrusted; I had a divil of a time recreating this for comparison purposes from the original but, when comparing it to my own WIP I'm noticing 3 minor differences. The first one is as a result of fixing the imperfection found in the middle of the original which I think we can let slide. The other 2 look to be as a result of removing a couple of seemingly excess points which you can see below (yours in red) - the first one is the top edge of the bottom right block and the second the bottom edge of the topmost block.

As I'm comparing with my own work, though, I'll let one of the other @simple-icons/maintainers throw another eye over this before requesting any changes as the problems may, in fact, be with my version.

**Comment by PeterShaggyNoble** [comment]
As there has been no further feedback on this, I'm going to merge this in as-is, we can update it at a later date if someone feels it's necessary.

Thanks for picking up my slack on this one, @adamrusted 👍🏻

**Pull request merged**